brion/spikeReport.h

```diff
@@ -279,6 +287,12 @@ class SpikeReport : public boost::noncopyable
      */
     BRION_API void write( const Spikes& spikes );
 
+    /**
+     * @return Whether the report supports seek to t < getCurrentTime() or not.
+     * @version 2.0
+     */
+    BRION_API bool supportsBackwardSeek() const;
+
```

Comment:
This function is really needed. Before you even seek you have to decide whether you're going to store the collected spikes or not because moving forward works in all cases but you still want to keep the spikes for future random access. Waiting until seek returns false (or throws) to start storing means that you may have already lost all the data (e.g. if you let playback move forward until the end and you want to loop it over afterwards).